.circleci/config.yml

```diff
@@ -221,6 +135,9 @@ executors:
       image: 'windows-server-2019-vs2019:stable'
       resource_class: windows.2xlarge
       shell: bash.exe
+  linux-docker:
+    docker:
+      - image: zjay437/rocksdb:0.5
```

Comment:
Will there be a command or doc on how to update this?

Reply:
I added comment for that, and will update the internal wiki for that.